source/_integrations/hassio.markdown

```diff
@@ -51,6 +53,10 @@ For Home Assistant OS Supervisor provides following binary sensors:
 | ------- | ------------------ | ----------- |
 | Update Available | no | Whether there is an update available for OS
 
+## Update entities
+
+For all your installed add-ons, Home Assistant Core, Home Assistant Supervisor and for the Home Assistant Operating System (if you are running that), this integration will provide [update](/integrations/update) entities that provide information about bending updates, and will allow you to update to them.
```

Comment:
```suggestion
For all your installed add-ons, Home Assistant Core, Home Assistant Supervisor, and for the Home Assistant Operating System (if you are running that), this integration will provide [update](/integrations/update) entities that provide information about pending updates, and will allow you to update to them.
```

Reply:
So we are not bending updates? 😂

Reply:
<img src="https://media3.giphy.com/media/tNC2rod1uTrdC/giphy.gif"/>